openlr_dereferencer/maps/a_star/__init__.py

```diff
@@ -2,14 +2,19 @@
 Provides the shortest_path(map, start, end) -> List[Line] function, which
 finds a shortest path between two nodes.
 """
-from typing import AbstractSet, List, Dict, Optional, Callable
-from queue import PriorityQueue
-from ..abstract import MapReader, Line, Node
-from .tools import heuristic, reconstruct_path, find_minimum, LRPathNotFoundError, tautology
+from typing import List, Optional, Callable
+from heapq import heapify, heappush, heappop
+from collections import namedtuple
+from ..abstract import Node, Line
+from .tools import heuristic, LRPathNotFoundError, tautology
+
+
+# Named tuples to represent a priority queue item and its score
+PQItem = namedtuple('PQItem', ['score', 'node', 'line', 'previous'])
```

Comment:
Maybe nitpicking, but to be consistent over the project, it would be cool to use typing.NamedTuple instead here.

Reply:
Ah.  I wasn't aware of this.  Sounds good.